# Nurtlesim Parameters
stddev_linear: 0.00                            # The standard deviation of the linear twist noise
stddev_angular: 0.00                           # The standard deviation of the angular twist noise
slip_min: -0.00                                # The minimum wheel slip for the slip noise
slip_max: 0.00                                 # The maximum wheel slip for the slip noise

# Obstacle Parameters
obstacles_coordinate_x: [3.0, 2.0, -2.0, -3.0]  # The x coordinate of the obstacles
obstacles_coordinate_y: [3.0, -1.0, -3.0, 1.0]  # The y coordinate of the obstacles               
obstacles_radius: 0.0762                        # The radius of the cardboard tubes [m]
max_visable_dist: 5.0                           # The maximum distance beyond which tubes are not visible [m]